### ISSUE TITLE
Simplify icon/asset/image/file terminology in the icon uploader

### ISSUE THREAD

**Issue by javierarce** [CartoDB/cartodb#11507]
The icon upload modal shows a bunch of different terms (asset, image, file, icon, upload) to refer to the same thing. We should simplify the wording to use the best term, which in my opinion could be just "icon".

Here are the strings we are currently using: 

```
 "add-asset": {
        "maki-icons": "Maki Icons",
        "modal-desc": "or just use our nice selection",
        "modal-title": "Select marker image",
        "pin-icons": "Pin Icons",
        "set-image": "Set image",
        "simple-icons": "Simple Icons",
        "upload-file": "Upload file",
        "upload-asset": "Upload asset",
        "upload-image": "Upload your image",
        "your-uploads": "Your uploads"
      },
```
cc: @noguerol @xavijam

**Comment by noguerol** [comment]
+1 for "icon"
